src/guarneri/instrument.py

```diff
@@ -150,13 +170,23 @@ def make_devices(self, defns: Sequence[Mapping], fake: bool):
         """
         # Validate all the defitions
         for defn in defns:
-            Klass = self.device_classes[defn["device_class"]]
+            try:
+                Klass = self.device_classes[defn["device_class"]]
```

Comment:
Here's the point where `defn["device_class"]` could be added to the `self.device_classes` dictionary using a [`dynamic_import()`](https://github.com/BCDA-APS/apstools/blob/e9780ed838d95aa726ffda14e40f95b5e0d0da76/apstools/utils/misc.py#L198).

Reply:
With `dynamic_import()`, it is possible to use an empty `device_classes` dict: `Instrument({})`.

Reply:
Sounds good. Tagging https://github.com/spc-group/guarneri/issues/6 here for reference.